samples/Calling/src/app/views/CallCompositeContainer.tsx

```diff
@@ -71,14 +73,26 @@ export const CallCompositeContainer = (props: CallCompositeContainerProps): JSX.
   }
 
   return (
-    <CallComposite
-      adapter={adapter}
-      fluentTheme={currentTheme.theme}
-      rtl={currentRtl}
-      callInvitationUrl={callInvitationUrl}
-      formFactor={isMobileSession ? 'mobile' : 'desktop'}
-      options={options}
-    />
+    <Stack className={mergeStyles({ width: '100%', height: '100%' })}>
```

Comment:
Is this intended to be included in the sample?

Reply:
Nope. I was testing their scenario. Nice catch!